airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/utils.py

```diff
@@ -10,7 +10,12 @@ def datetime_to_secs(dt: datetime.datetime) -> int:
     return calendar.timegm(dt.utctimetuple())
 
 
-def string_to_date(d: str, f: str = "%Y-%m-%d") -> datetime.date:
+def string_to_date(d: str, f: str = "%Y-%m-%d", old_format=None) -> datetime.date:
```

Comment:
once `DATE_FORMAT` is moved here, we can directly mention it's value as default for `f: str`, instead of hadcode.

Reply:
`f: str = "%Y-%m-%d"` >> `f: str = DATE_FORMAT `

Reply:
Done